hydroflow_lang/src/graph/ops/anti_join_multiset.rs

```diff
@@ -0,0 +1,163 @@
+    ports_out: None,
+    properties: FlowProperties {
+        deterministic: FlowPropertyVal::Preserve,
+        monotonic: FlowPropertyVal::No,
+        inconsistency_tainted: false,
+    },
+    input_delaytype_fn: |idx| match idx {
+        PortIndexValue::Path(path) if "neg" == path.to_token_stream().to_string() => {
+            Some(DelayType::Stratum)
+        }
+        _else => None,
+    },
+    write_fn: |wc @ &WriteContextArgs {
+                   root,
+                   context,
+                   hydroflow,
+                   op_span,
+                   ident,
+                   inputs,
+                   op_inst:
+                       OperatorInstance {
+                           generics:
+                               OpInstGenerics {
+                                   persistence_args,
+                                   ..
+                               },
+                           ..
+                       },
+                   ..
+               },
+               diagnostics| {
+
+        let persistences = match persistence_args[..] {
+            [] => [Persistence::Tick, Persistence::Tick],
+            [a] => [a, a],
+            [a, b] => [a, b],
+            _ => unreachable!(),
+        };
+
+        let mut make_antijoindata = |persistence, side| {
+            let antijoindata_ident = wc.make_ident(format!("antijoindata_{}", side));
+            let borrow_ident = wc.make_ident(format!("antijoindata_{}_borrow", side));
+            let (init, borrow) = match persistence {
+                Persistence::Tick => (
+                    quote_spanned! {op_span=>
+                        #root::util::monotonic_map::MonotonicMap::<_, #root::rustc_hash::FxHashSet<_>>::default()
+                    },
+                    quote_spanned! {op_span=>
+                        &mut *#borrow_ident.get_mut_clear(#context.current_tick())
+                    },
+                ),
+                Persistence::Static => (
+                    quote_spanned! {op_span=>
+                        #root::rustc_hash::FxHashSet::default()
+                    },
+                    quote_spanned! {op_span=>
+                        &mut *#borrow_ident
+                    },
+                ),
+                Persistence::Mutable => {
+                    diagnostics.push(Diagnostic::spanned(
+                        op_span,
+                        Level::Error,
+                        "An implementation of 'mutable does not exist",
+                    ));
+                    return Err(());
+                }
+            };
+            Ok((antijoindata_ident, borrow_ident, init, borrow))
+        };
+
+        let (neg_antijoindata_ident, neg_borrow_ident, neg_init, neg_borrow) =
+                make_antijoindata(persistences[1], "neg")?;
+
+        let write_prologue = quote_spanned! {op_span=>
+            let #neg_antijoindata_ident = #hydroflow.add_state(std::cell::RefCell::new(
```

Comment:
`persistences[0]` is never referenced in this code so how can `'static` persistence on the pos side work?

Reply:
should also add a test for that case too.